### ISSUE TITLE
Support for Technicolor 

### ISSUE THREAD

**Issue by TheKSPMan** [hack-technicolor/hack-technicolor#282]
**General info**

My gateway is currently running 17.1.8066-0001006-20211207165728 from Telia in Lithuania.

- Product: MediaAccess TG789vac v2
- ISP: Telia Lithuania
- Countries: -
- Commercial name from the ISP: Technicolor DGA0122
- Board: VANT-6

**Did anybody ever manage to get root on that device already?**

*Select all applicable*
- [ ] No, as far as I know it has never been rooted

Just want to update the firmware on this router. On the bottom it says 2015 and I hope to get the latest firmware available.

**Comment by LuKePicci** [comment]
Yeah the firmware is quite old but the described root strategies will probably work. Just try and let us known. When done you can easily update to MST firmware 17.2 which is slightly newer

**Comment by antnks** [comment]
There is no newer firmware for this model. The firmware is the same as for tg389ac
Samba hack patched, tch-exploit works

**Comment by forkiesassds** [comment]
> There is no newer firmware for this model. The firmware is the same as for tg389ac Samba hack patched, tch-exploit works

Any idea on how to make tch-exploit work? I've tried using it with no success (stuck after DHCP communication, nothing after). I have tried hacking the router before to make it function due to a misconfiguration making it WAN only (we only have DSL) as it was a replacement unit (old one broke due to thunder). After no avail we had it fixed by Telia maintenance team. However they somehow managed to import old port forwarding configuration from the original router. Trying to change anything in relation to open ports causes this error.
![Screenshot_20240803_012612](https://github.com/user-attachments/assets/511efa51-e6dc-4336-b054-ef1160c27d20)
Thus why I am interested in hacking the router again.

The router runs firmware `17.1.8066-0001006-20211207165728`, same as OP. However there is an older firmware `17.1.7992-0001013-20180515151649`, which I'm unsure how to obtain.

**Comment by Sidicer** [comment]
> There is no newer firmware for this model. The firmware is the same as for tg389ac Samba hack patched, tch-exploit works

Is there no newer firmware than `17.1.8066-0001008-20211207165407` for `tg389ac` or can `tg389ac` receive version `17.2` from some other router?

Question is because I have a `GANT-1 TG389ac` with `17.1.8066-0001008-20211207165407` firmware which I can see has not been dumped yet (https://github.com/hack-technicolor/hack-technicolor/tree/master/torrents/gant-1) and I was wondering if I should atleast dump `bank_1` and `bank_2` and upload it. Or would that be not useful?

I am trying to use `tch-exploit` on it, but it doesn't move past first `BOUND` so I'm not sure it works on `*.8066`? How feasible would it be to downgrade to `*.7992` to then try and get root (which in turn of course would rid us of the possibility of dumping `*.8066`)?

**Comment by antnks** [comment]
Yes, you should downgrade and then you can exploit it.
